configs/deploy/detection_onnxruntime_dynamic.py

```diff
@@ -0,0 +1,39 @@
+onnx_config = dict(
```

Comment:
same codebase config and onnx config appear in multiple configs, could simply by making  base static and dynamic configs.